pkg/machine/qemu/machine.go

```diff
@@ -482,6 +482,10 @@ func (v *MachineVM) SSH(name string, opts machine.SSHOptions) error {
 	}
 
 	cmd := exec.Command("ssh", args...)
+	if logrus.GetLevel() == logrus.DebugLevel {
+		fmt.Printf("Executing: ssh %v\n", args)
```

Comment:
Should this be logrus.Debugf?

Reply:
I guess in this case the getlevel is not required as there is no heavy compute inside the of statement

Reply:
if you prefer, i think it would work.

Reply:
> Should this be logrus.Debugf?

Yes.